src/BookingConflictConditionFactory.php

```diff
@@ -63,16 +76,53 @@ class BookingConflictConditionFactory implements FactoryInterface
      */
     protected $exprBuilder;
 
+    /**
+     * A list of non-blocking booking statuses.
+     *
+     * @since [*next-version*]
+     *
+     * @var array|stdClass|Traversable
+     */
+    protected $nonBlockingStatuses;
+
     /**
      * Constructor.
      *
      * @since [*next-version*]
      *
-     * @param object $exprBuilder The expression builder.
+     * @param object                     $exprBuilder         The expression builder.
```

Comment:
Why not hint [`ExpressionBuilderInterface`][]? Maybe even typehint.

[`ExpressionBuilderInterface`]: https://github.com/RebelCode/rcmod-wp-cqrs/blob/develop/src/Expression/Builder/ExpressionBuilderInterface.php#L15

Reply:
Because we are going to revise expression builder in the future, most likely not use that interface, and to remain consistent with everything else that needs an expression builder.

This _will_ sound rude, but I want you to not read it _that_ way.

You've brought this up numerous times already. And every time I say the same thing. Please don't bring up expression builder again :P At least not until we actually have time to revise it.

Reply:
It's also not even part of this PR, except for the indentation 😆